cfg/cfg.go

```diff
@@ -101,6 +110,18 @@ func init() {
 		"Output events that are usually sent Che master instead of regular logs to imitate what a user can see."+
 			"`false` by default. Needed for testing and debugging purposes",
 	)
+	flag.StringVar(
+		&RegistryAddress,
+		"registry-address",
+		"",
+		"Address of registry from which to retrieve meta.yamls when only plugin IDs are specified.",
+	)
+	flag.BoolVar(
+		&DownloadMetas,
+		"download-metas",
+		false,
+		"Download plugin metadata from registry instead of process already-downloaded metas.",
```

Comment:
Other messages don't have trailing slashes. Please, align new ones

Reply:
Not entirely sure what you mean, but I've removed `.` from the end of these messages.